Gemfile.lock

```diff
@@ -65,7 +65,9 @@ GEM
     marcel (0.3.2)
       mimemagic (~> 0.3.2)
     method_source (0.9.0)
-    mimemagic (0.3.2)
+    mimemagic (0.3.10)
```

Comment:
update `mimemagic` for install problem
ref https://stackoverflow.com/questions/66919504

Reply:
We have an issue here with Mercel depending on the yanked version of Mimemagic. If you upgrade Mercel to v1.0.0 or newer this removed the dependency on Mimemagic.

https://github.com/rails/marcel/releases
https://rubygems.org/gems/mimemagic/versions

Reply:
Thank you for your advise! I've upgraded Mercel with 4e54b45